include/service/entities/plugin_service.js

```diff
@@ -1071,6 +1130,7 @@ module.exports = function PluginServiceModule(pb) {
         var details = null;
         var site = plugin.site || GLOBAL_SITE;
         var cached_plugin = PLUGIN_INIT_CACHE[plugin.uid] || null;
+        var autoSync = pb.config.settings.autoSync || false;
```

Comment:
It is reasonable to assume that there will be a value for the setting.  I wouldn't default it.